web/docs/tql2/operators/chart_area.md

```diff
@@ -0,0 +1,56 @@
+# chart_area
+
+Chart values on a X-Y area chart.
+
+```tql
+chart_area x=any, y=any, [x_min=any, x_max=any, resolution=duration, x_log=bool, y_log=bool, group=any, position=string]
+```
+
+## Description
+
+Chart values on X-Y area chart.
```

Comment:
This is missing a word. But in any case, can we write something more useful here? I feel like the description could be a good place to explain how the chart is derived from the incoming events (at least in the simple case where just `x` and `y` is specified, i.e., each event becomes a point in the chart), and that the charts can be viewed on [app.tenzir.com](app.tenzir.com).

Reply:
Good point!